questionReply/questionReply-war/src/main/java/org/silverpeas/components/questionreply/control/QuestionReplySessionController.java

```diff
@@ -612,10 +578,11 @@ public void relaunchRecipients() throws QuestionReplyException {
 
       // recupere la liste de tous les experts du domaine de classement de la
       // question
-      ContainerPositionInterface position = getSilverContentIdPosition();
+      SearchContext position = getSilverContentIdPosition();
       if (position != null && !position.isEmpty()) {
-        List<Integer> liste =
-            containerContext.getSilverContentIdByPosition(position, listeInstanceId);
+        //List<Integer> liste = containerContext.getSilverContentIdByPosition(position, listeInstanceId);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACommentedOutCodeLine)